sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -847,6 +847,12 @@ object SQLConf {
       .intConf
       .createWithDefault(UnsafeExternalSorter.DEFAULT_NUM_ELEMENTS_FOR_SPILL_THRESHOLD.toInt)
 
+  val SUPPORT_QUOTED_REGEX_COLUMN_NAME = buildConf("spark.sql.parser.quotedRegexColumnNames")
+    .doc("When true, quoted Identifiers (using backticks) in SELECT statement are interpreted" +
```

Comment:
@janewangfb You can do something like:

    scala> val df = Seq((1, 2), (2, 3), (3, 4)).toDF("a", "b")
    df: org.apache.spark.sql.DataFrame = [a: int, b: int]
    scala> df.groupBy("a", "b").agg(df.col("*")).show
    +---+---+---+---+
    |  a|  b|  a|  b|
    +---+---+---+---+
    |  2|  3|  2|  3|
    |  1|  2|  1|  2|
    |  3|  4|  3|  4|
    +---+---+---+---+

So I guess you can also do something like:

    scala> df.groupBy("a", "b").agg(df.colRegex("`...`"))